dashboard/modules/healthz/healthz_agent.py

```diff
@@ -50,4 +50,4 @@ async def run(self, server):
 
     @staticmethod
     def is_minimal_module():
-        return True
+        return False
```

Comment:
I think this is not a minimal module since it depends on aio http server. cc @rkooo567 can you confirm?

Reply:
yeah I think you are right